tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1146,6 +1150,7 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark:
   xfail:
     reason: "Headroom pool size not supported."
     conditions:
+      - "hwsku in ['Force10-S6100'] and topo_type in ['t1-64-lag']"
       - "hwsku not in ['Arista-7060CX-32S-C32', 'Celestica-DX010-C32', 'Arista-7260CX3-D108C8', 'Force10-S6100', 'Arista-7260CX3-Q64', 'Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8']"
```

Comment:
these should should be skipped not xfail (as is done for testQosSaiHeadroomPoolSize

Reply:
updated to "skip"